.github/workflows/test.yml

```diff
@@ -209,6 +209,84 @@ jobs:
             echo "=> go test CGO_ENABLED=1"
             env CGO_ENABLED=1 go test -shuffle=on -v -count=10 ./...
 
+            if [ -z "$(echo ${{matrix.go}} | grep '^1.1')" ]; then
+              echo "=> go test race"
+              go test -race -shuffle=on -v -count=10 ./...
+            fi
+  netbsd:
```

Comment:
Let's add to build tests for NetBSD in other platforms (e.g. Can we build a NetBSD binary in FreeBSD?)

```
            # Check cross-compiling NetBSD binaries.
            env GOOS=netbsd GOARCH=amd64 go build -v ./...
            env GOOS=netbsd GOARCH=arm64 go build  -v ./...
```

Reply:
I'm not sure if it's possible. We'd need to explicitly enable cgo since it's a cross compile. I'll try once the build in NetBSD is working